skrub/_multi_agg_joiner.py

```diff
@@ -0,0 +1,417 @@
+"""
+The MultiAggJoiner extends AggJoiner to multiple auxiliary tables.
+"""
+import numpy as np
+from sklearn.base import BaseEstimator, TransformerMixin
+from sklearn.utils.validation import check_is_fitted
+
+from skrub import _join_utils
+from skrub._agg_joiner import AggJoiner
+from skrub._dataframe._namespace import is_pandas, is_polars
+from skrub._utils import _is_array_like, atleast_2d_or_none
+
+
+class MultiAggJoiner(BaseEstimator, TransformerMixin):
+    """Extension of the AggJoiner to multiple auxiliary tables.
+
+    Apply numerical and categorical aggregation operations on the `cols`
+    to aggregate, selected by dtypes. See the list of supported operations
+    at the parameter `operations`.
+
+    If `cols` is not provided, `cols` are all columns from `aux_tables`,
```

Comment:
could you reword this a little bit to make it clear that there is one list of columns, one aux key for each aux table?